src/components/editor/property/InputLookupQA.jsx

```diff
@@ -142,11 +120,42 @@ export const renderTokenFunc = (option, tokenProps, idx) => {
 //  based on values in propertyTemplate.valueConstraint.useValuesFrom
 //  and the lookupConfig for the URIs has component value of 'lookup'
 const InputLookupQA = (props) => {
+  const [authorities, setAuthority] = useState([])
+
+  const search = (query) => {
+    const currentAuthorities = []
+    const resultPromise = getSearchResults(query, props.propertyTemplate)
```

Comment:
Not the greatest but I think speeding up/refactoring so that we are not using `Promise.all` is a refactoring work. The QA search speed on the Sinopia side hasn't changed with this PR.